server/metrics/global_status.go

```diff
@@ -65,10 +80,25 @@ type RequestStatus struct {
 	// bytes are the sum of document and index bytes written, and the read bytes in this case would be the data
 	// update or delete request where there may be a scan depending on the query filter.
 	writeBytes int64
-	// Some operations have a fixed cost, and it should be calculated at the iterator level
-	ddlDropUnits   int64
+	// Type of search request 0 - api search, 1 - collection search (used internally)
+	searchRequestType int
+	// One drop operation (for example drop collection) is one drop unit
+	ddlDropUnits int64
+	// One create operation (for example one create collection) is one create unit
 	ddlCreateUnits int64
+	// One schema update operation is one update unit
 	ddlUpdateUnits int64
+	// TODO: implement bytes written for search
+	// One search request or getting the next page from search is one search unit
+	searchUnits int64
+	// One collection search request or getting the next page from search is one collection search unit
+	collectionSearchUnits int64
+	// One create search index is one create search index unit
+	searchCreateIndexUnits int64
+	// One drop search index is one create search index unit
+	searchDropIndexUnits int64
+	// Deleting one document from a search index is one delete document unit
+	searchDeleteDocumentUnits int64
```

Comment:
we meed to also add `searchWriteBytes`

Reply:
It's on a TODO item, the linter caught that it's unused and made me remove the actual field.